### DISCUSSION TITLE
Obtain expense details for "OFA" type

### DISCUSSION THREAD

**Discussion by yao657355700** [allegro/allegro-api#8174] [Suggestions - rejected]
1. Obtain expense details for "OFA" type
Because we need to have a clear understanding of the true cost of each order, we need to download this data
2. Obtain product size and weight information

**Comment by PrzemyslawLukanowski** [comment]
We have added order.id in the GET /billing/billing-entries, but currently we don't have plans to add information about product size and weight.
